aggregator/src/datastore.rs

```diff
@@ -718,6 +704,94 @@ impl<C: Clock> Transaction<'_, C> {
         )?)
     }
 
+    /// Add an aggregator authentication token to the task with the given `task_id`.
+    pub async fn add_aggregator_authentication_token(
+        &self,
+        task_id: &TaskId,
+        token: &AuthenticationToken,
+    ) -> Result<(), Error> {
+        let params: &[&(dyn ToSql + Sync)] = &[&task_id.as_ref()];
```

Comment:
I had to keep one `params` let binding in `add_collector_authentication_token`, where I run futures through `try_join!()`, since using a temporary wouldn't last past the statements constructing the futures.